### DISCUSSION TITLE
CSSBattle's LEARN - Level #23

### DISCUSSION THREAD

**Discussion by mchxyz** [cssbattle/forum#306] [General]
Hey there!

I've been trying to complete level #23 of CSSBattle's LEARN but unfortunately the webpage doesn't let me continue after inputting what I believe is the right value (100px) in order to create a circle. 

<div></div>

<style>

  div { 

    width: 200px;

    height: 200px;

    background-color: purple;

    margin: 20px;

    
    border-radius: 100px;

  }

</style>

P.S.: I've tried with a % as well and it didn't work. 

So, is this a bug or am I just stuck?

You can check it out at https://cssbattle.dev/learn/23

Thanks in advance for your reply!

**Comment by aarthvaen** [comment]
It seems that the level doesn't accept a border-radius value in px. However, using percentage, i.e. `border-radius: 50%;` seems to overcome the issue (on my end, at least).

**Comment by mchxyz** [comment]
Thank you! I tried using a border radius of 100% as well using both Safari and Chrome browsers (either from a desktop or a phone) and it didn’t work for me. 

![7C563323-6CEA-4F3F-9AB2-2EE0AC6676C7](https://user-images.githubusercontent.com/111715559/189483973-219fd523-fff3-4e40-ac18-12a9ec516221.jpeg)
![7D29E923-54F6-47FB-9D6D-FBA27C143D52](https://user-images.githubusercontent.com/111715559/189483975-8be34e4a-204c-4a90-af63-f148a7780b60.jpeg)
![2C093DE3-0EF9-4C31-987E-C75BE4641ED6](https://user-images.githubusercontent.com/111715559/189483976-2ae0738c-6cf6-42fa-96a7-9e2acab13144.jpeg)

**Reply by chinchang** [reply]
100px is 50% of `<div>`'s width. So border-radius should 50% :)

**Reply by chinchang** [reply]
<img width="728" alt="CleanShot 2022-09-10 at 19 37 35@2x" src="https://user-images.githubusercontent.com/379918/189487061-4669daa4-786c-4239-9c0f-34eb4787a0cc.png">

**Comment by mchxyz** [comment]
Awesome! Thank you.
